extension/content/contexts/environment.js

```diff
@@ -315,7 +353,9 @@ export async function login(dispatch, selectedKey, environment) {
     url: webAuth.client.buildAuthorizeUrl({
       state,
       nonce,
+      ...(details.interactive === false ? { prompt: "none" } : {}),
```

Comment:
unsure about this pattern. it saves a bunch of lines of code but maybe it's not super clear to read?

alternative:
```js
const buildUrlOptions = {
  none,
  state,
}

if (details.interactive === false) {
  buildUrlOptions.prompt = "none";
}

webAuth.client.buildAuthorizedUrl(buildOptions);
```

Reply:
as an aside... i did actually use this trick in that react devtools patch that you merged:
https://github.com/mozilla-extensions/normandy-devtools/commit/33e503ebde706ee419f03eed25fdaf9f370dda8f#diff-11e9f7f953edc64ba14b0cc350ae7b9dR48

should I undo that?

Reply:
I don't think it's worth changing there.